tests/integration/chroot.py

```diff
@@ -147,10 +147,13 @@ def _generate_ssh_key(self):
         """ Generate new SSH key for integration test """
         logger.info("Generating new SSH key for integration tests.")
         ssh_key_path = self.config["ssh"]["ssh_key_filepath"]
-        keyfp = RemoteClient.generate_key_pair(
-            filename = ssh_key_path,
-        )
-        logger.info("SSH key for integration tests generated.")
+        if not os.path.isfile(ssh_key_path):
```

Comment:
In my opinion, this if  statement should be part of the `generate_key_pair` function itself. Maybe, we could add an additional parameter, that triggers an exception if the key already exists (e.g. `force`)